src/util/std_expr.h

```diff
@@ -2126,6 +2126,11 @@ inline not_exprt &to_not_expr(exprt &expr)
   return ret;
 }
 
+// Create a lexicographic less-than relation between two variables.
```

Comment:
I see value in having this as a utility function but this is not the place for it.  `std_expr.h` is for the `irept` subclasses that give interfaces for expressions.  It's a bit of a weird case though that doesn't obviously fit into any of the other parts of `util/`.  Maybe it could have it's own file?

Reply:
Thanks a lot for your comment! I will discuss the issue with Saswat.

Reply:
@kroening Can I have your input? As Martin pointed out, `std_expr.h` may not be a desirable place for the function `create_lexicographic_less_than`. However, creating a separate file just for the function `create_lexicographic_less_than` is not an ideal solution, either.

Reply:
As @kroening suggested, I decided to put the function `create_lexicographic_less_than` in the file `contracts.cpp`.